docs/wetboek.md

```diff
@@ -73,7 +73,7 @@
     2. Verwerking van de in beslag genomen goederen (behalve voertuigen) geschiedt volgens richtlijn van Politie;
 2. Wanneer een voertuig in beslag wordt genomen vanwege een strafbaar feit:
     1. Kan deze maximaal drie (3) dagen worden vastgehouden voor onderzoek;
-    2. Zal deze, nadat bewijsbaar is aangetoond dat het voertuig onderdeel was van een strafbaar feit, uitgekocht kunnen worden op basis van de ernst van het strafbaar feit;
+    2. Zal deze, nadat bewijsbaar is aangetoond dat het voertuig onderdeel was van een strafbaar feit, uitgekocht kunnen worden op basis van de ernst van het strafbaar feit, 24 uur nadat de inbeslagname is geregistreerd in de politiesystemen;
```

Comment:
Hoe weet de speler dit?

Reply:
In principe is dit gewoon het tijdstip van inbeslagname (of enkele minuten daarna), maar het is lastig om dat te bewijzen. Het moment van registratie in politiesystemen is het enige dat wordt opgeslagen.

Reply:
check